modules/cpmso-yaml-sample-cr.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-sample-cr_{context}"]
+= Sample YAML for a control plane machine set custom resource
```

Comment:
Would love input on this in terms of what is useful/important to know (though this one I feel pretty ok about)